docker/jmeter-worker/Dockerfile

```
@@ -1,4 +1,5 @@
-FROM hellofreshtech/kangal-jmeter:base
+ARG version="latest"
```

pls upper case for args

```suggestion
ARG VERSION="latest"
```